### PR TITLE
Adding new Perception_Statistics.ipynb notebook 

### PR DESCRIPTION
**Pull request by JonathanHUnity**
# Peer Review Information
This PR adds a new public notebook called Perception_Statistics which contains stats and visualizations for the annotations and metrics supported by the Perception package out of the box. For the most part it follows the same style as the SynthDet notebook. One significant way it differs is that it supports each type of data being missing from the dataset, allowing it to be used in any configuration of Perception and not just when all labelers are present.

It also makes a few changes which fixes errors when loading a dataset that has no annotations.

Feel free to suggest where I should add tests or documentation

# Pull Request Check List

- [x] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

File: datasetinsights/data/simulation/captures.py [hunk 1]
```diff
@@ -122,14 +123,20 @@ def _load_annotations(self, data_root, version):
...
+                    self.TABLE_NAME,
+                    version,
+                    record_path="annotations",
+                    meta="id",
+                    meta_prefix="capture.",
+                )
+            except KeyError:
+                annotation = pd.DataFrame(
+                    {"annotation_definition": [], "capture.id": []}
+                )
```
Comment (BlairLee) [inline datasetinsights/data/simulation/captures.py]: Can you add a unittest for this exception?

**Comment by BlairLee** [comment]
With all those updates, are we going to rename the `SynthDet_Statistics.ipynb` to `Perception_Statistics.ipynb`?

**Comment by JonathanHUnity** [comment]
> With all those updates, are we going to rename the `SynthDet_Statistics.ipynb` to `Perception_Statistics.ipynb`?

This is adding a new notebook that is more general to Perception. SynthDet_Statistics.ipynb has a bunch of statistics that are specific to SynthDet, so I think it's best to keep both. What do you think?

File: datasetinsights/data/simulation/captures.py (hunk 1 above)
Comment (JonathanHUnity) [reply datasetinsights/data/simulation/captures.py]: Sure. do you want a test that `load_table` throws this exception in certain cases or a test that Captures(...) properly handles this case?

Comment (BlairLee) [reply datasetinsights/data/simulation/captures.py]: Would be better to test whether `Captures._load_annotations` can handle this case.

**Comment by JonathanHUnity** [comment]
@BlairLee Where should we add documentation about the new notebook, if anywhere?

**Comment by BlairLee** [comment]
> @BlairLee Where should we add documentation about the new notebook, if anywhere?

I believe it should be in the end-to-end documentation. (https://github.com/Unity-Technologies/SynthDet/pull/20/files)

**Review by BlairLee — APPROVED** [review]

**Pull request merged**